pandas/tests/test_base.py

```diff
@@ -683,6 +683,10 @@ def test_factorize(self):
 
     def test_duplicated_drop_duplicates(self):
         # GH 4060
+
+        import warnings
+        warnings.simplefilter('always')
```

Comment:
Ah, this is for testing ``tm.assert_produces_warning`` in a loop. Should remove?